WordPress/Classes/ViewRelated/System/Coordinators/MySitesCoordinator.swift

```diff
@@ -56,14 +44,9 @@ class MySitesCoordinator: NSObject {
         splitViewController.restorationIdentifier = MySitesCoordinator.splitViewControllerRestorationID
         splitViewController.presentsWithGesture = false
         splitViewController.setInitialPrimaryViewController(navigationController)
-        splitViewController.dimsDetailViewControllerAutomatically = !FeatureFlag.newNavBarAppearance.enabled
+        splitViewController.dimsDetailViewControllerAutomatically = false
         splitViewController.tabBarItem = navigationController.tabBarItem
-
-        if Feature.enabled(.newNavBarAppearance) {
-            splitViewController.wpPrimaryColumnWidth = .default
-        } else {
-            splitViewController.wpPrimaryColumnWidth = .narrow
-        }
+        splitViewController.wpPrimaryColumnWidth = .default
```

Comment:
same here, isn't `.default` the default value? can we remove this line?

Reply:
Will check, thanks!

Reply:
Done